src/ripple/nodestore/impl/codec.h

```diff
@@ -319,7 +319,7 @@ nodeobject_compress (void const* in,
     using std::runtime_error;
     using namespace nudb::detail;
 
-    std::size_t type = 1;
+    std::size_t const codecType = 1;
```

Comment:
Consider `constexpr ` and moving to line ~466.

Reply:
Fixed in 42d4b481a `Misc fixups in shadow warnings commit`